src/content-blocks/BlockButtons/BlockButtons.tsx

```diff
@@ -1,23 +1,46 @@
 import classnames from 'classnames';
 import React, { FunctionComponent } from 'react';
 
-import { Button, ButtonProps } from '../../components/Button/Button';
+import { Button } from '../../components/Button/Button';
+import { ButtonType } from '../../components/Button/Button.types';
 import { ButtonToolbar } from '../../components/ButtonToolbar/ButtonToolbar';
-import { AlignOptions, DefaultProps } from '../../types';
+import { AlignOptions, ButtonAction, DefaultProps, IconName } from '../../types';
+
+export interface ButtonProps extends DefaultProps {
+	active?: boolean;
+	ariaLabel?: string;
+	arrow?: boolean;
+	autoHeight?: boolean;
+	block?: boolean;
+	disabled?: boolean;
+	icon?: IconName;
+	label?: string;
+	action: ButtonAction;
+	size?: 'small';
+	title?: string;
+	type?: ButtonType;
+}
 
 export interface BlockButtonsProps extends DefaultProps {
 	elements: ButtonProps[];
 	align?: AlignOptions;
+	buildAndNavigate: (buttonAction: ButtonAction) => void;
```

Comment:
in de client zou ik deze functie idd `buildAndNavigate` maar in de components zou ik deze functie gewoon `navigate` noemen.